### PR TITLE
Co simulate defocus

### PR DESCRIPTION
**Pull request by cossorzano**

### CODE REVIEW

File: xmipp3/tests/test_protocol_simulate_ctf.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+    def testSimulateCTF(self):
+        importedVolume = self.runImportVolume(self.volume, 1)
+        projectionsGallery = self.runCreateGallery(importedVolume)
+
+        # Check default defoci values
+        projectionsCTFDefault = self.runSimulateCTF(projectionsGallery)
+        self.assertTrue(projectionsCTFDefault,
+                        "There was a problem with particles output")
+        self.assertTrue(projectionsCTFDefault.getSize() == 1647,
```
Comment (DStrelak) [inline xmipp3/tests/test_protocol_simulate_ctf.py:83]: assertEqual

File: xmipp3/tests/test_protocol_simulate_ctf.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                        "There was a problem with particles output")
+        self.assertTrue(projectionsCTFDefault.getXDim() == 60,
+                        "Unexpected particle size in output particles")
+        self.assertTrue(projectionsCTFDefault.getSamplingRate() == 1,
+                        "Unexpected sampling rate in output particles")
+        for projection in projectionsCTFDefault.iterItems():
+            defocusU = projection.getCTF().getDefocusU()
+            defocusV = projection.getCTF().getDefocusV()
+            if defocusU != defocusV:
+                raise Exception('DefocusU and DeofcusV are not equal but they should be')
```
Comment (DStrelak) [inline xmipp3/tests/test_protocol_simulate_ctf.py:93]: use asserts

File: xmipp3/tests/test_protocol_simulate_ctf.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            else:
+                if defocusU < 5000 or defocusU > 25000:
+                    raise Exception('DefocusU and DefocusV values are outside the specified range:'
+                                    '[%f, %f]' % (5000, 25000))
+
+        # Check different defoci range
+        projectionsCTFNewRange = self.runSimulateCTF(projectionsGallery, 1000, 3000)
+        self.assertTrue(projectionsCTFNewRange,
```
Comment (DStrelak) [inline xmipp3/tests/test_protocol_simulate_ctf.py:101]: copy&paste code

File: xmipp3/tests/test_protocol_simulate_ctf.py (hunk 1 above)
Comment (cossorzano) [reply xmipp3/tests/test_protocol_simulate_ctf.py:83]: Corrected

File: xmipp3/tests/test_protocol_simulate_ctf.py (hunk 2 above)
Comment (cossorzano) [reply xmipp3/tests/test_protocol_simulate_ctf.py:93]: Corrected

File: xmipp3/tests/test_protocol_simulate_ctf.py (hunk 3 above)
Comment (cossorzano) [reply xmipp3/tests/test_protocol_simulate_ctf.py:101]: Removed

**Pull request merged**